nntrainer/tensor/manager.cpp

```diff
@@ -440,9 +446,13 @@ Manager::requestOutputs(const GraphNode &node,
                         const std::vector<TensorDim> &outputs_dim) {
   const auto &exec_order = node.getExecutionOrder();
   std::vector<unsigned int> var_exec_order(
-    {std::get<0>(exec_order), std::get<2>(exec_order)});
+    {std::get<0>(exec_order)}); /** forwarding */
+  if (node.getType() == ActivationLayer::type)
+    /** TODO: if removing this reduces memory consumption, resolve this */
+    var_exec_order.push_back(std::get<2>(exec_order)); /** calcDerivative */
```

Comment:
I wonder what the bug was related to this?
does var only need to be saved for the output?

Reply:
activation layer depends on its output to reduce the computation, but a general layer depends on its input.

Reply:
I see